src/app/v5_challenges/challenge.manage.user.controller.js

```diff
@@ -82,6 +86,23 @@ module.controller('v5challenge.ManageUserController', ['$scope', '$rootScope', '
             }
         };
 
+        /**
+         * gets the e-mail by user id.
+         * @param {string} userId the user id.
+         */
+        $scope.getEmail = function (userId) {
+            var user = _.find($scope.usersEmails, function (x) {
+                if (x.userId == userId) {
+                    return x;
+                }
+            });
```

Comment:
As we already using lodash here, we might compact this code to
```js
var user = _.find($scope.usersEmails, { userId });
```

Reply:
did the same for getRole